source/agora/utils/WellKnownKeys.d

```diff
@@ -1170,23 +1170,25 @@ static immutable CommonsBudget = KeyPair(PublicKey(Point([49, 189, 206, 20, 131,
 
-    Note that despite mining for a few days, NODE0, NODE1, NODE8, NODE9 were
-    not found.
-
 *******************************************************************************/
 
-/// vald2: boa1xpvald2ydpxzl9aat978kv78y5g24jxy46mcnl7munf4jyhd0zjrc5x62kn
-static immutable NODE2 = KeyPair(PublicKey(Point([89, 223, 181, 68, 104, 76, 47, 151, 189, 89, 124, 123, 51, 199, 37, 16, 170, 200, 196, 174, 183, 137, 255, 219, 228, 211, 89, 18, 237, 120, 164, 60])), SecretKey(Scalar([40, 122, 190, 63, 119, 203, 168, 25, 99, 173, 205, 177, 194, 117, 61, 201, 34, 111, 32, 177, 208, 131, 42, 123, 234, 196, 116, 61, 121, 14, 102, 2])));
-/// vald3: boa1xrvald3zmehvpcmxqm0kn6wkaqyry7yj3cd8h975ypzlyz00sczpzhsk308
-static immutable NODE3 = KeyPair(PublicKey(Point([217, 223, 182, 34, 222, 110, 192, 227, 102, 6, 223, 105, 233, 214, 232, 8, 50, 120, 146, 142, 26, 123, 151, 212, 32, 69, 242, 9, 239, 134, 4, 17])), SecretKey(Scalar([249, 97, 58, 139, 241, 57, 94, 224, 245, 226, 65, 242, 24, 2, 44, 139, 229, 2, 61, 93, 97, 53, 117, 209, 198, 20, 254, 213, 226, 180, 254, 14])));
-/// vald4: boa1xrvald4v2gy790stemq4gg37v4us7ztsxq032z9jmlxfh6xh9xfak4qglku
-static immutable NODE4 = KeyPair(PublicKey(Point([217, 223, 182, 172, 82, 9, 226, 190, 11, 206, 193, 84, 34, 62, 101, 121, 15, 9, 112, 48, 31, 21, 8, 178, 223, 204, 155, 232, 215, 41, 147, 219])), SecretKey(Scalar([98, 226, 48, 109, 185, 208, 123, 178, 255, 129, 73, 131, 35, 237, 45, 205, 165, 127, 59, 193, 148, 70, 235, 102, 79, 137, 73, 116, 53, 66, 231, 15])));
-/// vald5: boa1xzvald5dvy54j7yt2h5yzs2432h07rcn66j84t3lfdrlrwydwq78cz0nckq
-static immutable NODE5 = KeyPair(PublicKey(Point([153, 223, 182, 141, 97, 41, 89, 120, 139, 85, 232, 65, 65, 85, 138, 174, 255, 15, 19, 214, 164, 122, 174, 63, 75, 71, 241, 184, 141, 112, 60, 124])), SecretKey(Scalar([221, 126, 197, 133, 227, 104, 135, 143, 81, 160, 228, 54, 200, 241, 175, 123, 2, 170, 165, 2, 7, 127, 122, 59, 7, 35, 45, 180, 140, 49, 166, 11])));
-/// vald6: boa1xrvald6jsqfuctlr4nr4h9c224vuah8vgv7f9rzjauwev7j8tj04qee8f0t
-static immutable NODE6 = KeyPair(PublicKey(Point([217, 223, 183, 82, 128, 19, 204, 47, 227, 172, 199, 91, 151, 10, 85, 89, 206, 220, 236, 67, 60, 146, 140, 82, 239, 29, 150, 122, 71, 92, 159, 80])), SecretKey(Scalar([108, 48, 31, 186, 38, 25, 57, 33, 60, 210, 196, 151, 102, 61, 167, 8, 100, 128, 104, 220, 204, 27, 39, 67, 101, 67, 43, 119, 30, 25, 30, 4])));
-/// vald7: boa1xzvald7hxvgnzk50sy04ha7ezgyytxt5sgw323zy8dlj3ya2q40e6elltwq
-static immutable NODE7 = KeyPair(PublicKey(Point([153, 223, 183, 215, 51, 17, 49, 90, 143, 129, 31, 91, 247, 217, 18, 8, 69, 153, 116, 130, 29, 21, 68, 68, 59, 127, 40, 147, 170, 5, 95, 157])), SecretKey(Scalar([118, 108, 243, 157, 25, 205, 32, 124, 50, 230, 106, 83, 17, 206, 84, 190, 145, 205, 57, 39, 114, 49, 53, 82, 78, 242, 202, 178, 115, 229, 63, 8])));
+// val2: boa1xzval2a3cdxv28n6slr62wlczslk3juvk7cu05qt3z55ty2rlfqfc6egsh2 - SAQXRDHTWME4GUIVNYCKPN433VJ4BJP2L2T7UWHGSSW47VFC67EQFY3S
```

Comment:
Good! I think "val2" looks better and "vald2". 
But this change should be another commit, I think.

Reply:
Problem is it is needed for the sorted outputs in the freeze transaction of the GenesisBlock. If it is a separate commit to change the keys then all the unit tests have to be fixed twice.

Reply:
Ah, I see. Ty.

Reply:
In fact I am have some problem with it so will split it out into separate commit as you suggest.